### DISCUSSION TITLE
Sigma Correlations Draft

### DISCUSSION THREAD

**Discussion by Neo23x0** [SigmaHQ/sigma-specification#86] [General]
So many people ask me about the correlations draft that I'd like to add it here for discussion

https://onedrive.live.com/view.aspx?resid=3454E59DF98D7D65!7485&ithint=file%2cdocx&authkey=!ADb97TgRX9Fr4xQ

Changes so far: 
- relations will not be based on filenames but on rule identifiers

**Comment by frack113** [comment]
More information here : https://github.com/SigmaHQ/sigma-specification

**Comment by nasbench** [comment]
Correlation specs have been released as part of 2.0. See https://blog.sigmahq.io/introducing-sigma-specification-v2-0-25f81a926ff0
